cacts/tests/test_utils.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from cacts.utils import expect, run_cmd, run_cmd_no_fail, expand_variables, evaluate_commands, str_to_bool, is_git_repo
+
+def test_expect():
+    with pytest.raises(RuntimeError):
+        expect(False, "This is an error message")
+
+def test_run_cmd():
+    stat, output, errput = run_cmd("echo Hello, World!")
+    assert stat == 0
+    assert output == "Hello, World!"
+
+def test_run_cmd_no_fail():
+    output = run_cmd_no_fail("echo Hello, World!")
+    assert output == "Hello, World!"
+
```

Comment:
I would also test that
```suggestion
with pytest.raises(RuntimeError):
   run_cmd_no_fail("false")
   
```
to check that this throws.